zephyr/ra/portable/src/rp_sci_b_spi/rp_sci_b_spi.c

```diff
@@ -0,0 +1,163 @@
+/*
+* Copyright (c) 2020 - 2025 Renesas Electronics Corporation and/or its affiliates
+*
+* SPDX-License-Identifier: BSD-3-Clause
+*/
+
+/***********************************************************************************************************************
+ * Includes
+ **********************************************************************************************************************/
+#include "r_sci_b_spi.h"
+#include "r_sci_b_spi_cfg.h"
+
+/***********************************************************************************************************************
+ * Macro definitions
+ **********************************************************************************************************************/
+#define SCI_B_SPI_PRV_DATA_REG_MASK               (0xFFFFFF00)
+#define SCI_B_SPI_PRV_RDAT_MASK                   (0xFFU)
+
+/***********************************************************************************************************************
+ * Typedef definitions
+ **********************************************************************************************************************/
+
+/***********************************************************************************************************************
+ * Private function declarations.
+ **********************************************************************************************************************/
+
+/***********************************************************************************************************************
+ * Functions
+ **********************************************************************************************************************/
+
+/*******************************************************************************************************************//**
+ * @addtogroup SCI_B_SPI
+ * @{
+ **********************************************************************************************************************/
+
+/*******************************************************************************************************************//**
+ * Enable polling data transfer.
+ *
+ * @param      p_api_ctrl           Pointer to the control structure.
+ *
+ * @retval     FSP_SUCCESS          Read operation successfully completed.
+ * @retval     FSP_ERR_ASSERTION    One of the following invalid parameters passed:
+ *                                  - Pointer p_api_ctrl is NULL
+ *
+ * @return     See @ref RENESAS_ERROR_CODES or functions called by this function for other possible return codes. This
+ *             function calls:
+ **********************************************************************************************************************/
+fsp_err_t RP_SCI_B_SPI_StartTransferPolling (spi_ctrl_t * const    p_api_ctrl)
```

Comment:
```suggestion
fsp_err_t RP_SCI_B_SPI_StartTransferPolling (spi_ctrl_t * const p_api_ctrl)
```
Please check and update similar cases